test/loader.js

```diff
@@ -0,0 +1,8 @@
+describe('Handlebars.helpers.loader', function() {
+
+    it('should return a div', function() {
```

Comment:
Is there any value in increasing the test coverage for the other branches here? I guess not if this is cut/paste and not new logic!?